tests/phpunit/tests/block-supports/duotone.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * Test the block WP_Duotone class.
+ *
+ * @package WordPress
+ */
+
+class WP_Duotone_Test extends WP_UnitTestCase {
```

Comment:
I don't know if this is the correct place for the test. We didn't have the old test that was removed in https://github.com/WordPress/gutenberg/pull/49103 and this is where it made sense to me...